express/public/main.js

```diff
@@ -92,14 +92,14 @@ var vm = new Vue({
 
           data.map(point => {
             labels.push(point.time);
-            tempData.push((point.weather.main.temp - 273.15).toFixed(2)); // Converting temperature from Kelvin to Celsius
-            humidData.push(point.weather.main.humidity.toFixed(2));
+            tempData.push(( (point.weather.currently.temperature - 32)*(5/9) ).toFixed(2)); // Converting temperature from Fahrenheit to Celsius
```

Comment:
Looks like you could add `units=si` to the API request and it would return celsius to save this